source/docs/appendices/migration-guides.md

```diff
@@ -20,6 +20,22 @@ These new features may be relevant for your existing application:
 - [A quickorder module has been created to order by SKU](/docs/advanced/features/quickorder.html)
 - Front-Commerce dependencies are now regularly (and automatically) updated using [Depfu](https://depfu.com/). Patches are automatically applied if the CI is green, minor versions are manually approved. We always review changelogs provided by Depfu. **You can review updates any time [by filtering Merge Requests tagged `depfu`](https://gitlab.com/front-commerce/front-commerce/-/merge_requests?scope=all&state=merged&label_name[]=depfu).**
 
+### Storybook 6
+
+Your styleguide is now powered by Storybook 6. 
+
+This induces several changes in your custom stories.
+
+- The organization of stories is now only manageable with `/`, please set your story name like this : `section/path/of/your/story/Component`
+- As knobs are now deprecated, we highly encourages you to use controls from now on. see [the official documentation for more details](https://storybook.js.org/addons/@storybook/addon-knobs)
+- The way to import markdown files in stories has changed, please update your code as below:
+```diff
+- import readme from "./readme.md";
++ import { html as readme } from "./readme.md";
+```
+
+Front-commerce provides a codemod to automatically migrate the hierarchy and readme imports. Run `npx jscodeshift -t scripts/codemod/transforms/storybook5-to-storybook6.js <module_path>` to make the necessary replacements in all .story.js files.
```

Comment:
> Run `npx jscodeshift -t scripts/codemod/transforms/storybook5-to-storybook6.js <module_path>` to make the necessary replacements in all .story.js files

You can replace the `npx jscodeshift -t scripts/codemod/transforms/storybook5-to-storybook6.js <module_path>` with `front-commerce codemod -t storybook5-to-storybook6`, If the `storybook5-to-storybook6` codemod [exports the extensions](https://gitlab.com/front-commerce/front-commerce/-/blob/d0f09f27add2a6da50a7c427338deec0eee419d8/scripts/codemod/transforms/storybook5-to-storybook6.js#L67), then it will only run on `*..story.js` files. and it will scan folders based on the .front-commerce.js configuration.

Reply:
command changed